quick-start/src/main/ui/app/traces/traces.component.html

```diff
@@ -1,54 +1,10 @@
-<div layout-padding layout="column" layout-align="center center">
-  <h1>Traces</h1>
-
-  <div class="search-box" flex layout-align="center center" layout="row">
-    <form (submit)="doSearch()" flex layout-align="center center" layout="row">
-      <input type="search" name="searchText" [(ngModel)]="searchText" placeholder="Enter you search query here...">
-      <button type="submit" mdl-colored="primary" mdl-button mdl-button-type="raised">Search</button>
-    </form>
-  </div>
-  <div *ngIf="!loadingTraces" layout="column" flex layout-align="center center" class="full-width">
-    <app-pagination [start]="searchResponse.start"
-          [total]="searchResponse.total"
-          [pageLength]="searchResponse.pageLength"
-          (pageChanged)="pageChanged($event)"></app-pagination>
-    <div *ngIf="!loadingTraces && (!traces || traces.length === 0)">No Traces</div>
-    <div class="mdl-grid results">
-      <div class="mdl-cell mdl-cell--2-col">
-        <app-facets
-          [facets]="searchResponse.facets"
-          [(activeFacets)]="activeFacets"
-          (activeFacetsChange)="updateFacets($event)"></app-facets>
-      </div>
-      <div class="mdl-cell mdl-cell--10-col">
-        <table left-align
-          class="mdl-data-table mdl-js-data-table mdl-data-table--selectable mdl-shadow--2dp"
-          *ngIf="traces && traces.length > 0">
-          <thead>
-            <tr>
-              <th class="mdl-data-table__cell--non-numeric">Type</th>
-              <th class="mdl-data-table__cell--non-numeric">Identifier</th>
-              <th class="mdl-data-table__cell--non-numeric">Job ID</th>
-              <th class="mdl-data-table__cell--non-numeric">Created</th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr *ngFor="let trace of traces" (click)="showTrace(trace.traceId)" [ngClass]="{'error' : trace.hasError }">
-              <td><i class="mdi" [ngClass]="getIconClass(trace)"></i> {{trace.flowType}}</td>
-              <td class="mdl-data-table__cell--non-numeric">{{trace.identifier}}</td>
-              <td class="mdl-data-table__cell--non-numeric">{{trace.jobId}}</td>
-              <td>{{trace.created | datePipe}}</td>
-            </tr>
-          </tbody>
-        </table>
-      </div>
-    </div>
-  </div>
-  <div *ngIf="loadingTraces" flex layout="column" layout-align="center center">
-    <h3>Loading Traces...</h3>
-    <div class="center-text">
-      <mdl-spinner single-color active></mdl-spinner>
-    </div>
-  </div>
-
-</div>
+<app-traces-ui
```

Comment:
The idea is to remove UI artifacts from here to a shared place. Please remove this html file and use template string interpolation instead with this markup, as well as remove please original scss and reference to it here https://github.com/marklogic/marklogic-data-hub/blob/f3a44a429a165146fd003ce7cd70b09220b623dc/quick-start/src/main/ui/app/traces/traces.component.ts#L12

Reply:
As far as I understand, It actually server the sample purpose. I will follow the standard though.

Reply:
Yes, we do not change logic, just structure, and some additional refactoring.